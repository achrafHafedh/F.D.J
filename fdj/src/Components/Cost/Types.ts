export type CostProps = {
  cost: string | number;
};
