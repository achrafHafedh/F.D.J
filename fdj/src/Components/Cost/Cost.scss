.cost {
  padding: 5px 10px;
  border: 1px solid black;
  text-align: center;
}
