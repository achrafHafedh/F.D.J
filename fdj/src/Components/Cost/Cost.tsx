import { FunctionComponent } from "react";

import { CostProps } from "./Types";

import "./Cost.scss";

const Cost: FunctionComponent<CostProps> = ({ cost }) => {
  return <div className="cost">Mise totale {cost}</div>;
};

export default Cost;
