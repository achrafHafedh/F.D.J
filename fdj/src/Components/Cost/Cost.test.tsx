import React from "react";
import { render } from "@testing-library/react";

import Cost from "./Cost";

describe("<Cost/>", () => {
  it("should render Cost with value", () => {
    const { container } = render(<Cost cost={2.5} />);
    expect(container).toMatchSnapshot();
  });
});
