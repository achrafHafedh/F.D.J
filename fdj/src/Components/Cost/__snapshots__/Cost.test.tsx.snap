// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Cost/> should render Cost with value 1`] = `
<div>
  <div
    class="cost"
  >
    Mise totale 
    2.5
  </div>
</div>
`;
